lib/src/configuration.cc

```diff
@@ -358,6 +359,17 @@ void Configuration::defineDefaultValues()
                     Types::Int,
                     5) } });
 
+    defaults_.insert(
+        Option { "allowed-keepalive-timeouts",
+                 Base_ptr { new Entry<int>(
+                    "allowed-keepalive-timeouts",
+                    "",
+                    lth_loc::translate("Number of consecutive keepalive response "
```

Comment:
Not sure about making this public, as using it also requires knowing that keepalives happen every 15 seconds with a 5 second timeout. Tempted to make it hidden as a default, because hopefully people won't need to tweak it.

@kylog any further thoughts?